[
{
"cve_id": "CVE-2024-29189",
"patch_date": [
"2024-03-22 08:32:20+01:00",
"2024-03-21 17:45:48+01:00"
],
"patch_files": [
"doc/changelog.d/1076.fixed.md",
"src/ansys/geometry/core/connection/product_instance.py",
"doc/changelog.d/1077.added.md"
],
"earliest_patch_date": "2024-03-21 17:45:48+01:00",
"earliest_vuln_date": "2023-08-18 08:31:49+00:00",
"vuln_files": [
"src/ansys/geometry/core/connection/product_instance.py"
],
"impact": {},
"cwes": [],
"fix_duplicate_type": false,
"fix_duplicate_types": [],
"intro_duplicate_type": false,
"intro_duplicated_types": [],
"file_fixed_before_fixed_again": false,
"files_fixed_before_fixed_again": [],
"file_fixed_before_vuln_again": false,
"files_fixed_before_vuln_again": []
}
]